lib/shoulda/matchers/independent/delegate_method_matcher.rb

```diff
@@ -202,14 +222,19 @@ def matches?(subject)
           subject_has_delegating_method? &&
             subject_has_delegate_object_reader_method? &&
             subject_delegates_to_delegate_object_correctly? &&
-            subject_handles_nil_delegate_object?
+            subject_handles_nil_delegate_object? &&
+            subject_handles_private_delegation?
         end
 
         def description
           string =
             "delegate #{formatted_delegating_method_name} to the " +
             "#{formatted_delegate_object_reader_method_name} object"
 
+          if expects_private_delegation?
+            string << " privately"
+          end
```

Comment:
Do you mind changing to single quotes? Rubocop raised at this line in the CI

```suggestion
          if expects_private_delegation?
            string << ' privately'
          end
```

Reply:
Sure thing. Force-pushed updated implementation commit.